root/static/scripts/common/utility/sortByEntityName.js

```diff
@@ -0,0 +1,16 @@
+/*
+ * @flow strict
+ * Copyright (C) 2020 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import sortBy from 'lodash/sortBy';
+
+export default function sortByEntityName(
+  entities: $ReadOnlyArray<CoreEntityT>,
+): $ReadOnlyArray<CoreEntityT> {
+  return sortBy(entities, [entity => entity.name.toLowerCase()]);
```

Comment:
If there's only one sort function, you can pass it directly rather than as a single-item list. Although...I also wonder if we can just perform this sorting on the server instead, where it does `@entities = nsort_by { $_->id == $target ? 0 : 1 } @entities;` in `Controller::Role::Merge` to be precise, by sticking a `sort_by` in there.
```suggestion
  return sortBy(entities, entity => entity.name.toLowerCase());
```

Reply:
For now I made the suggested change on the JS.